notebooks/MAST/plotting_quasar_observations/plotting_quasar_observations.ipynb

```diff
@@ -0,0 +1,866 @@
+{
```

Comment:
<p>This could use some more explanation about what you notice from the table above (oh, I have a lot of repeated observations with the same instrument and filters, I wonder how many of each) and then how you'd like to summarize. The "exptime" and "expmax" columns in particularl could use more explanation.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/ScienceBetter/notebooks-stsci/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>